sdks/python/apache_beam/examples/snippets/transforms/aggregation/groupby_test.py

```diff
@@ -78,178 +76,193 @@ def normalize_kv(k, v):
 # For documentation.
 NamedTuple = beam.Row
 
+
 def check_groupby_expr_result(grouped):
-  assert_that(
-      grouped | beam.MapTuple(normalize_kv),
-      equal_to([
-          #[START groupby_expr_result]
-          ('s', ['strawberry']),
-          ('r', ['raspberry']),
-          ('b', ['banana', 'blackberry', 'blueberry']),
-          #[END groupby_expr_result]
-      ]))
+  pass  # TODO: Uncomment asserts
```

Comment:
Actually, since they're structured this way (sorry, I thought the asserts were inline, these tests are structured a bit oddly with the code and asserts is a separate file and a third line that links them), maybe just return here to reduce churn.

Reply:
Ha, I initially put in early returns, but didn't want to fight possible flake8 issues about unreachable code since my IDE was giving me a hard time about it. I realize now flake8 doesn't complain about unreachable code.